packages/web3.js-plugin-fee-estimation/tsconfig.json

```
@@ -0,0 +1,15 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist",
+    "baseUrl": "./src",
+    "strict": true,
```

```suggestion
    "strict": true,
    "verbatimModuleSyntax": true,
    "noUncheckedIndexedAccess": true,
```
VerbatimModuleSyntax forces us to add `type` to imports
noUncheckedIndexAccess makes ts scrictor when it comes to index types.

These are the 2 most important missing config options but more optional strict rules can be found in this file https://github.com/evmts/evmts-monorepo/blob/main/tsconfig/base.json